kernelci/api/latest.py

```diff
@@ -51,6 +51,19 @@ def whoami(self) -> dict:
     def password_hash(self, password: str) -> dict:
         return self._post('/hash', {'password': password}).json()
 
+    def change_password(self, username: str, current: str, new: str) -> dict:
+        """Change a password for a given user"""
+        return self._post(
+            '/password',
+            {
+                'current_password': {'password': current},
+                'new_password': {'password': new},
+            },
+            {
+                'username': username,
+            },
```

Comment:
Now I'm wondering if having `/password/<username>` would be a better design for the API endpoint.  But that's just an implementation detail, wanted to share it here as a thought.

Reply:
(or `/<username>/password`)

Reply:
> (or `/<username>/password`)

+1 on this.

Reply:
I think the original idea was that the username wasn't required if the query was using an API token for authentication.  But since it requires the old password we don't rely on a token so the user endpoint makes more sense.

Reply:
> > (or `/<username>/password`)
> 
> +1 on this.

Do you mean a new API path or should it be a part of `/user`, i.e. `/user/<username>/password`?

I can adjust it to also accept API token instead of only the old password.